### DISCUSSION TITLE
Why does RouteChangeAnnouncement in the basic example have an “id”?

### DISCUSSION THREAD

**Discussion by silvenon** [remix-run/remix#763] [Q&A] [answered]
I love the `RouteChangeAnnouncement` component that we get in the basic example, I didn't know about this a11y problem before or how to fix it, so I learned something! I'm curious, though, what is the purpose of the `id` on the `<div>` element?

https://github.com/remix-run/remix/blob/ec349f0ab628699f69ad0dba27d08bfe759772f9/examples/basic/app/root.tsx#L230

**Comment by matmilbury** [comment, marked as answer]
It seems that the `id` attribute can be removed.

**Reply by silvenon** [reply]
Yeah, it's not being referred to anywhere else in the codebase. I guess I should've opened a PR rather than asking this question.

**Reply by matmilbury** [reply]
https://github.com/remix-run/remix/pull/826
